central/alert/service/service_impl.go

```diff
@@ -510,16 +538,19 @@ func countAlerts(alerts []*storage.ListAlert, groupByFunc func(*storage.ListAler
 	return
 }
 
-func getMapOfAlertCounts(alerts []*storage.ListAlert, groupByFunc func(alert *storage.ListAlert) []string) (groups map[string]map[storage.Severity]int) {
+func getMapOfAlertCounts(alerts []search.Result, groupByFunc func(alert search.Result) []string) (groups map[string]map[storage.Severity]int) {
 	groups = make(map[string]map[storage.Severity]int)
 
 	for _, a := range alerts {
 		for _, g := range groupByFunc(a) {
 			if groups[g] == nil {
 				groups[g] = make(map[storage.Severity]int)
 			}
-
-			groups[g][a.GetPolicy().GetSeverity()]++
+			if len(a.Matches[search.Severity.String()]) == 0 {
+				continue
+			}
+			severity, _ := strconv.Atoi(a.Matches[search.Severity.String()][0])
```

Comment:
same comment as above regarding the field path

Reply:
??